index.html

```diff
@@ -406,6 +407,64 @@ <h2>Ballerina in Action</h2>
 }
                                     </code>
                                 </pre>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+             </div>
+             <div id="seventhTab" class="tab-pane fade in active"  >
+                 <div class="row" style="background-color: #f5f5f5;">
+                     <div class="col-lg-12 col-md-12 col-sm-12 focusPane " style="padding:20px;">
+                         <div class="FullCode" style="background-color: #f5f5f5;">
+                             <div class="highlight">
+                               <pre class="language-ballerina basic ballerina">
+                                <code class="language-ballerina ballerina" style="font-size: 12px !important;">
+import ballerina/http;
+import ballerina/sql;
+import ballerinax/mysql;
+import ballerinax/mysql.driver as _;
+
+configurable string host = ?;
+configurable int port = ?;
+configurable string user = ?;
+configurable string password = ?;
+configurable string database = ?;
+
+type Album record {|
+    string id;
+    string title;
+    string artist;
+    decimal price;
+|};
+
+final mysql:Client dbClient = check new (host, user, password, database, port);
+
+service / on new http:Listener(8080) {
+    resource function get albums() returns Album[]|sql:Error? {
```

Comment:
I wouldn't use `sql:Error` as the return type.

Reply:
Fixed in e8e8dc8